CONTRIBUTING.md

```diff
@@ -444,17 +444,23 @@ When choosing `minor` or `major`, make sure your PR targets the `next` branch in
 
 The description of the changeset should follow the these guidelines:
 
-- Our changesets should be about _user-facing_ changes. Internal changes don't
-  need changesets.
-- Use the past tense when describing what you did, e.g. "Added new feature", "Fixed edge case".
-- Use the present tense when describing Biome behavior, e.g. "Biome now supports ...".
+- Our changesets should be about **user-facing changes**. Internal changes don't
+  need changesets. For example, if you refactored some code but the user-facing
+  behavior didn't change, you don't need a changeset.
+- **Be concise and clear.** Changesets are not documentation and they are not test cases.
+  They should give a **quick overview** of what changed, allowing the reader to dig deeper if they want to. A good changeset is usually between 1 and 3 sentences long. **Longer changesets indicate to the user that they should pay more attention to the change, but that is not always the case.** Consider the impact of your change on the user when writing the changeset. The key is to provide just enough information for the user to understand the change without overwhelming them with details.
+  - For a *new* lint rule, show an example of an invalid case in a code block. If it _really_ helps demonstrate the rule, you can also show a valid case.
+  - For a *change in an existing rule*, clearly demonstrate what is now considered invalid that wasn't before, or vice versa. If it helps communicate the change greatly, show both invalid and valid cases.
```

Comment:
To be honest, I think we should always provide a code snippet. It's the best way to communicate a rule change. What do you think?

Reply:
It depends on if you make a distinction between inline code snippets and code blocks. Let's take this changeset for example: 

> Added the `noDistractingElements` lint rule for HTML. The rule enforces that no distracting elements are used.
> 
> Invalid:
> 
> ```html
> <marquee /> 
> <blink />
> ```
> 
> Valid:
> 
> ```html
> <div />
> ```

It is true that it's really clear. However, it can be simplified to a single line and still provide the same value:

> Added the noDistractingElements lint rule for HTML. The rule enforces that no distracting elements like `<marquee>` or `<blink>` are used.

Snippets are still being used, but they are inline and not blocks.